sdgym/datasets.py

```diff
@@ -122,11 +121,6 @@ def get_available_datasets():
 
 
 def _get_available_datasets(modality, bucket=None, aws_key=None, aws_secret=None):
-    if modality not in MODALITIES:
```

Comment:
Should we still have a check to make sure `modality` isn't some random string?

Reply:
Very good catch, I shouldn't have deleted `MODALITIES` in this file, it's unrelated to the one in `synthesizers/base.py`.